source/features/dim-bots.css

```diff
@@ -0,0 +1,51 @@
+/*
+READ BEFORE EDITING. Some selectors match the same items ("hovered" selector vs non-hovered); make sure to update them in tandem.
+
+In PR lists we can hide the meta just by acting on `max-height` + `overflow: hidden`
+In Commit lists however this is not possible because they are tables, so we need to hack around `margin-bottom`
+
+Transitions are used to delay the "hover" status so it's not too annoying when moving the mouse over a list of dimmed items.
+*/
+.rgh-dim-bot.commit .commit-title,
+.rgh-dim-bot.commit .commit-meta,
+.rgh-dim-bot.commit .commit-links-cell summary,
+.rgh-dim-bot.commit .commit-links-cell .btn {
+	transition: 0.1s 0.3s;
```

Comment:
Titles are dimmed and meta info/buttons are hidden. This effect is undone with a slight delay on hover/focus, currently set to `0.3s`